src/main/frontend/components/header.cljs

```diff
@@ -39,8 +39,14 @@
   (let [_ (state/sub :auth/id-token)]
     (when-not config/publishing?
       (if (user-handler/logged-in?)
-        [:span.text-sm.font-medium (user-handler/email)]
-
+        (ui/dropdown-with-links
+         (fn [{:keys [toggle-fn]}]
+           [:a.button
+              {:on-click toggle-fn}
+            [:span.text-sm.font-medium (user-handler/email)]])
+         [{:title "Logout"
```

Comment:
It'd be useful to have Logout translated like login is below

Reply:
yes! I forget to add...